client/dashboard/profile-wizard/steps/theme/index.js

```diff
@@ -55,33 +56,64 @@ class Theme extends Component {
 	}
 
 	renderTheme( theme ) {
-		const { image, price, slug, title } = theme;
+		const { demo, has_woocommerce_support, image, slug, title } = theme;
```

Comment:
Heads up: `demo` is changing to `demo_url` in the API response.

Reply:
Thanks- updated!